app/views/hyrax/base/_items.html.erb

```diff
@@ -0,0 +1,46 @@
+<%# OVERRIDE: Hyrax 2.9.6 %>
+<h2><%= t('.header') %></h2>
+<%# OVERRIDE: Hyrax 2.9.6 to return all item ids %>
+<% array_of_ids = presenter.list_of_item_ids_to_display %>
+<% members = presenter.member_presenters_for(array_of_ids) %>
+<%# OVERRIDE: Hyrax 2.9.6 to hide jpg derivatives for non admin's/superadmin's %>
+<% page_count = presenter.total_pages %>
+
+<%
+  # OVERRIDE: Hyrax 2.9.6
+  unless current_user.try(:is_admin?) || current_user.try(:is_superadmin?)
```

Comment:
Curious why we are adding this?  My strong preference is to amend this to use `can?`. And even more so to amend the sole query to not include those to begin with.

Why?  Because these `is_admin?` are an anti-pattern. What is it about these roles that says to keep the `is_derived_ssi` value?

Reply:
actually thanks for catching that, i thought i changed it to

```rb
unless current_user&.has_role?(:admin, Site.instance) || current_user&.has_role?(:superadmin, Site.instance)
```

which comes from the [LV](https://github.com/scientist-softserv/louisville-hyku/commit/dee73fe67a10fbc055a9f1f880715f7e5683b201#diff-2dbfa1ee303b1781e4c78421ace29bf7248c922c798ea2ed38a76862d6253953)

I am curious about using `can?` instead though, I'm not too familiar with it